features/285-design-webhooks/events.md

```diff
@@ -58,16 +58,12 @@ information about the event:
 For BEFORE events about a user, following fields can be mutated:
 - `metadata`: user metadata
 - `verify_info`: verify info of user
-- `is_verified`: verified status of user
 - `is_disabled`: disbled status of user
 
 **NOTE**
 - The mutations would not affect authorization of current operation. e.g.
   disabling user through mutations would not fail an operation requiring
   non-disabled users.
-- `is_verified` is normally derived from `verify_info`. If it is mutated to be
```

Comment:
What about if we keep this, but add a boolean like `is_manually_verified` to `verify_info`? Than we can make the verification white list not a black magic?

If it is a bad idea, maybe just change this description to something like: `is_verified` is normally derived from `verify_info` and `is_manually_verifed` to keep the explaination (assuming it is relevant in context, I only looked at the track changes, not the whole doc)

Reply:
All keys in `verify_info` map are login IDs; moving `is_manually_verified` into `verify_info` would be very confusing.

This paragraph is describing the behavior of mutating `is_verified`. Since we do not allow mutating `is_verified` flag directly anymore, this paragraph is not relevant.